deploy/templates/deployment.yaml

```diff
@@ -55,6 +55,15 @@ spec:
           value: "{{ .Values.workloadIdentity.enabled }}"
         - name: WORKLOAD_IDENTITY_DISABLE_GLOBAL_SERVICE_ACCOUNT
           value: "{{ .Values.workloadIdentity.disableGlobalServiceAccount }}"
+        {{- if and (hasKey .Values "global") 
+            (hasKey .Values.global "azure")
```

Comment:
Not sure, will these values like `global`, `azure`, `extension` and `name` be added during the extension installation?

Reply:
Yes， this is what will be passed by the extension